views/search_results.erb

```diff
@@ -10,11 +11,11 @@
       </tr>
       <% flights.each do |flight| %>
         <tr>
-          <td><%= Time.parse(flight.takeoff_at).strftime('%I:%M %p') %></td>
-          <td><%= Time.parse(flight.land_at).strftime('%I:%M %p') %></td>
+          <td><%= TimeMath.datetime_for_airport(date, flight.takeoff_at, flight.orig).strftime('%I:%M %p %Z') %></td>
+          <td><%= TimeMath.datetime_for_airport(date, flight.land_at, flight.dest).strftime('%I:%M %p %Z') %></td>
```

Comment:
That's some solid advice, @burtlo. 🤜  🤛 

I smelled a Presenter in there, but haven't used them (knowingly?) a lot in the past. Also, it was late. Great idea for a refactor PR!